WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/quicklinksribbon/QuickLinkRibbonItemViewHolder.kt

```diff
@@ -14,6 +14,6 @@ class QuickLinkRibbonItemViewHolder(
         quickLinkItem.setText(item.label)
         quickLinkItem.setIconResource(item.icon)
         quickLinkItem.setOnClickListener { item.onClick.click() }
-        quickStartFocusPoint.setVisibleOrGone(item.showFocusPoint)
+        quickLinkRibbonItemQuickStartFocusPoint.setVisibleOrGone(item.showFocusPoint)
```

Comment:
Nitpick: Nice Improvement 👍🏼. How about we rename it to `quickLinkRibbonsItemQuickStartFocusPoint`?. 🤔 .

Reply:
Removed `Ribbon` and renamed to `quickLinkItemQuickStartFocusPoint` to match with `quickLinkItem` naming convention in the same class. Without `Ribbon` in the name, I think it is fine to keep `quickLink` singular.

In 7314eb4